src/util/ivmFactory.js

```
@@ -243,6 +243,11 @@ async function createIvm(code, libraryVersionIds, versionId, secrets, testMode)
     }),
   );
 
+  await jail.set('_credentials', function (...args) {
+    if (args.length == 0 || !credentials || !credentials[args[0]]) return 'ERROR';
```

What if we send multiple args ?

---

The first arg would be taken as the key, anything after that would be ignored

---

Should we raise an error as it is not allowed to send multiple keys

---

Also why make the function definition with (...args) then ?